Sources/Sentry/Public/SentryMessage.h

```diff
@@ -20,7 +20,7 @@ SENTRY_NO_INIT
 /**
  * Returns a @c SentryMessage with setting formatted.
  * @param formatted The fully formatted message. If missing, Sentry will try to interpolate the
- * message. It must not exceed 8192 characters. Longer messages will be truncated.
+ * message. The backend will truncate messages longer than 8192 characters.
```

Comment:
`m`: I am aware of the reasoning for changing this, but what would happen if a message with a huge payload, i.e. 1MB of data, is uploaded? Is there an upper limit which the backend will drop? Is "no limit" better than a super-high limit?

Reply:
The goal is to move this power to the backend so we can easily change it. The backend will drop envelopes exceeding a certain limit yes. This problem exists for all properties the user can set. We currently only truncate SentryMessage. There is an edge case when a user sets a very very long message and the backend will drop your message now because the SDK doesn't truncate it anymore, but you need to use a message exceeding 1MB.

Reply:
Thanks for clarifying.

Reply:
Thanks for double checking 🙏